src/fs.js

```diff
@@ -35,13 +35,13 @@ Fs.prototype = {
       return;
     }
 
+    relativeTo = relativeTo ? path.dirname(relativeTo) : process.cwd();
+    var dirs = relativeTo.split(path.sep);
     var foundFile;
-    var dirs = __dirname.split(path.sep);
```

Comment:
`__dirname` always resolved to the directory containing this file (`src/fs.js`).